src/reynir_correct/errfinder.py

```diff
@@ -313,7 +313,7 @@ def AðvörunHeldur(self, txt: str, variants: str, node: Node) -> AnnotationDict
             text="'{0}' er sennilega ofaukið".format(txt),
             detail="Yfirleitt nægir að nota 'en' í þessu samhengi.",
             original=orig_txt,
-            suggest="",     # Word should be deleted
+            suggest="DELETE",     # Word should be deleted
```

Comment:
Change: suggest="" if the token should be deleted; suggest=None if there is no suggestion

Reply:
Done